.github/workflows/build.yml

```
@@ -14,32 +14,14 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
+  check-labels:
     runs-on: ubuntu-24.04
-
+    outputs:
+      labels: ${{ steps.get-pr-labels.outputs.result }}
```

There is only at most one label:

```suggestion
      label: ${{ steps.get-pr-labels.outputs.result }}
```

---

Can be done in a `ci-build:skip` follow-up.